modular_skyrat/modules/modular_vending/code/wardrobes.dm

```diff
@@ -119,11 +119,9 @@
 		/obj/item/clothing/suit/toggle/deckard = 2,
 		/obj/item/clothing/suit/discoblazer = 2,
 		/obj/item/clothing/suit/toggle/labcoat/highvis = 2,
-		/obj/item/clothing/under/misc/discounder = 2,
```

Comment:
dont forget to add at least one of these back into the vendor under their new type; I say at least one because they're the exact same icons anyways (iirc), so they may be worth condensing into one anyways